omnigibson/simulator.py

```diff
@@ -489,8 +491,9 @@ def play(self):
             # Track whether we're starting the simulator fresh -- i.e.: whether we were stopped previously
             was_stopped = self.is_stopped()
 
-            # Minimize physics leakage by slowing down simulator significantly
-            with self.slowed(dt=1e-3):
+            # Run super first
+            # We suppress warnings from omni.usd because it complains about values set in the native USD
```

Comment:
sorry what does this mean?

Reply:
maybe add a bit more explicit comment here.

Reply:
done!